### PR TITLE
refactor: refine ampli config

### PR DESCRIPTION
**Pull request by dohaki**
Some small adjustments to the amplitude config.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
     configuration: {
       disableCookies: true,
+      logLevel: isProductionBuild ? Types.LogLevel.Error : Types.LogLevel.Debug,
+      trackingOptions: {
+        ipAddress: false,
+        carrier: false,
+        city: false,
+        region: false,
+        dma: false, // designated market area
+      },
```
Comment (mrice32) [inline src/App.tsx:26]: +1

File: src/App.tsx [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+  isProductionBuild,
+} from "utils";
 
 // Record an event when the application is loaded
 ampli.load({
   environment: amplitudeEnvironment,
+  disabled: !Boolean(amplitudeAPIKey),
   client: {
-    apiKey: amplitudeAPIKey || "",
+    apiKey: amplitudeAPIKey,
```
Comment (mrice32) [inline src/App.tsx:16]: Does the type matter here? I figured we would have to ensure this is a string before putting it here, but maybe not!

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by amateima — APPROVED** [review]

File: src/App.tsx (hunk 2 above)
Comment (dohaki) [reply src/App.tsx:16]: It shouldn't matter much according to the code here https://github.com/across-protocol/frontend-v2/blob/507eb56695fec2e4d942cc71a3cd85069aa25774/src/ampli/index.ts#L1067-L1074

**Pull request merged**